app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -132,9 +137,23 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         final Activity activity = getActivity();
 
         _hlEditor = view.findViewById(R.id.document__fragment__edit__highlighting_editor);
+        _editorHolder = view.findViewById(R.id.document__fragment__edit__editor_holder);
         _textActionsBar = view.findViewById(R.id.document__fragment__edit__text_actions_bar);
         _webView = view.findViewById(R.id.document__fragment_view_webview);
         _primaryScrollView = view.findViewById(R.id.document__fragment__edit__content_editor__scrolling_parent);
+        _primaryScrollView.getViewTreeObserver().addOnScrollChangedListener(() ->
+        {
+            if (_lineNumbersView.isLineNumbersEnabled()) {
+                final long time = System.currentTimeMillis();
+                if (time - _lineNumbersRefreshTime > 125) {
+                    _lineNumbersRefreshTime = time;
+                    _lineNumbersView.forceRefresh();
+                }
+            }
+        });
+        _lineNumbersView = view.findViewById(R.id.document__fragment__edit__line_numbers_view);
+        _lineNumbersView.setEditText(_hlEditor);
```

Comment:
IMO we should add the scroll change listener inside setEditText.

setEditText() should remove the listener from an the existing edit text when this is done